src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -500,10 +584,72 @@ private static void init() {}
          * Holds all fluids that are gaseous at room temperature.
```

Comment:
The one other fluid I wonder if its worth having is powdered snow, though I suspect that one is pretty uncommon (I have not even gotten around to including it in my mods). The fact it has a bucket form might be enough for some mod devs though.

Reply:
I would say powder snow is not a fluid as otherwise, sand can be considered a fluid if someone make a bucket for it which doesn't really make sense and that powder snow bucket places a block instead of fluid.

If I was doing a tech mod with fluid stuff, I would design my system to make people pump water around instead of powder snow. Then have a freezer block that takes in water and buckets and freezes the water into powder snow buckets and spit that out. That would be more logical imo.

So I'll leave out any sort of tag for powder snow fluid. Let mods decide if they want to add that tag on their own or not

Reply:
Gonna mark this as resolved for now but if we start seeing lots of mods with powder snow fluid for some odd reason, we can revisit this.